shell/devconfig.sh

```diff
@@ -88,17 +88,9 @@ DEPLOY_TO_DEV_ENVIRONMENT=local_development "$DIR/deploy-to-dev.sh" show | yq -r
     echo -e "$configData" | sed "s:/run/secrets/outreach.io:$configDir:g" |
       sed "s:/run/volumes/outreach.io:$volumeDir:g" >"$tmpFile"
 
-    # If the file already exists, then merge it with the new one.
-    # NOTE: We probably want to make this smarter and log when merging. I don't think
-    # that is feasible to do in bash, though....
-    if [[ -e $saveFile ]] && [[ ${saveFile##*.} =~ ^(yaml|yml)$ ]]; then
-      info_sub "$configFile (merged)"
-      yq --yaml-output -s '.[0] * .[1]' "$tmpFile" "$saveFile" >"$mergedFile"
-      mv "$mergedFile" "$saveFile"
-    else
-      info_sub "$configFile"
-      mv "$tmpFile" "$saveFile"
-    fi
+    info_sub "$configFile"
+    mv "$tmpFile" "$saveFile"
```

Comment:
As mentioned in our internal slack, this was a feature requested by people to allow local modifications to that configuration to be persisted across `make devserver` runs. Not sure how to fix this while retaining that functionality at the moment

Reply:
As a general qq: In this particular case can we not make this a flag that we pass in to the devenv command? that way we can support both... 

It feels like most users would not care for this merging. Does that seem reasonable? 
Something like --persist-config ?

Reply:
That's a question for @deregtd as the main person wanting this feature, I think.